packages/main/src/webComponents/ResponsivePopover/ResponsivePopover.stories.tsx

```diff
@@ -5,46 +5,60 @@ import { PlacementType } from '@ui5/webcomponents-react/lib/PlacementType';
 import { PopoverHorizontalAlign } from '@ui5/webcomponents-react/lib/PopoverHorizontalAlign';
 import { PopoverVerticalAlign } from '@ui5/webcomponents-react/lib/PopoverVerticalAlign';
 import { ResponsivePopover } from '@ui5/webcomponents-react/lib/ResponsivePopover';
-import React from 'react';
+import React, { useCallback, useRef } from 'react';
 
 export default {
   title: 'UI5 Web Components / ResponsivePopover',
   component: ResponsivePopover
 };
 
-export const generatedDefaultStory = () => (
-  <ResponsivePopover
-    allowTargetOverlap={boolean('allowTargetOverlap', true)}
-    headerText={text('headerText', 'Responsive Popover Header')}
-    horizontalAlign={select('horizontalAlign', PopoverHorizontalAlign, PopoverHorizontalAlign['Center'])}
-    initialFocus={text('initialFocus', '')}
-    modal={boolean('modal', false)}
-    noArrow={boolean('noArrow', false)}
-    placementType={select('placementType', PlacementType, PlacementType['Right'])}
-    verticalAlign={select('verticalAlign', PopoverVerticalAlign, PopoverVerticalAlign['Center'])}
-    footer={
-      <div
-        style={{
-          display: 'flex',
-          justifyContent: 'flex-end',
-          alignItems: 'center',
-          width: 'calc(100% - 1rem)',
-          height: '2.5rem'
-        }}
+export const generatedDefaultStory = () => {
+  const popoverRef = useRef();
+  const onButtonClick = useCallback(
+    (e) => {
+      debugger;
+      popoverRef.current.openBy(e.target);
```

Comment:
Even though `openBy` works here. Please use `open`
```suggestion
      popoverRef.current.open(e.target);
```